sites/docs/docs/guides/usql-migration-guide.md

```diff
@@ -0,0 +1,253 @@
+    - Replace the build command in your deployment provider to `npm run sources && npm run build:strict`
+    - See Netlify and Vercel docs in the Resources section
+
+## Special Situations to Migrate
+
+### Script Tags & Javascript References
+
+If you use a script tag on a markdown page, you will need to change any variable declarations of of `let` or `var` to `$:`.
+
+The reason for this change relates to how the new query engine updates query results - because queries run in the browser and can change based on input, you need to use a reactive variable declaration (that's what `$:` means), which can update itself when the data is instantiated, and when it changes.
+
+The good news is that most of what you currently include in a script tag should be able to be migrated to USQL by using parameterized SQL queries.
+
+For example:
+
+#### Before - Using `let`
+```html
+<script>
+    let filtered_data = my_query.filter(d => d.column === $page.params.my_variable);
+</script>
+
+<LineChart data={filtered_data}/>
+```
+
+#### After - Using `$:`
+```html
+<script>
+    $: filtered_data = my_query.filter(d => d.column === $page.params.my_variable);
+</script>
+
+<LineChart data={filtered_data}/>
+```
+
+#### Ideal - Refactor to SQL Query in USQL
+````html
+```filtered_data
+select * from my_table
+where column = '${params.my_variable}'
+```
+
+<LineChart data={filtered_data}/>
+````
+
+### Query Chaining
+Our migration steps do not take into account query chaining. In some cases, chained queries will continue to work as normal. In other cases, you will need to make adjustments.
+
+Query chaining is not supported in the `sources` directory, but is still supported in `queries` and on your markdown pages.
+
+This means that any query chains included in `sources` will need to be replaced with actual references to the tables you need.
+
+If you use the VS Code migration command, chained queries found on markdown pages are left on the page rather than being moved to the `sources` directory like other queries. This is because we assume that most chained queries are simple enough for the syntax of your source database to match with the DuckDB syntax they will need to move to. In some cases, the syntax will not line up and you will need to make an adjustment.
+
+### Evidence Plugins
+If your project includes an Evidence plugin (e.g., [Evidence Labs](https://labs.evidence.dev)):
+1. Find the `evidence.plugins.yaml` file in your `_legacy_project` folder and copy the line(s) containing the plugin(s) you're using
+2. Paste those lines into the `evidence.plugins.yaml` file in your new project
+3. Install the plugin(s) in your project. E.g.,:
+```shell
+npm install --save @evidence-dev/labs
+```
+
+### External Package Dependencies
+
+If your project includes external packages installed via npm, you will need to reinstall those packages so that they are reflected in your `package.json`
+
+```shell
+npm install <package-name>
+```
```

Comment:
````
### Monorepos
If your evidence project is installed in a monorepo then...
````

Reply:
Added this to the migration steps - ensure you open the evidence project folder specifically. not sure if there's anything else specific to a monorepo we should note?